### PR TITLE
feat: init postman collection

### PR DESCRIPTION
**Pull request by dogivalfer**
# Description

Started a new Postman collection with an accompanying environment to be used for automated API tests.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

The collection was tested directly through Postman pointing to the API running on the test environment.

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have already been accepted and merged

## Further comments

Jira Ticket: [FSADT2-50](https://apps.nrs.gov.bc.ca/int/jira/browse/FSADT2-50)

### CODE REVIEW

File: test/postman/nr-starting-app_env.postman_environment.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (RMCampos) [inline test/postman/nr-starting-app_env.postman_environment.json:1]: This file also looks like exactly the same as `nr-starting-api_test_collection.postman_collection.json`. Is that correct?

File: test/postman/nr-starting-api_test_collection.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,59 @@
+{
+	"info": {
+		"_postman_id": "db559797-d9ba-4bee-93f8-f33c3574ef63",
+		"name": "nr-starting-api test collection",
+		"description": "## Description\nThis collection contains the API tests of the starting API.\n## Coverage\nThe aim is to have a coverage of 100%, every request on the API should have a mathcing test in this collection.\n## What to test\n- Status code\n- Response body field validation\n- Response time (1s avg)",
```
Comment (RMCampos) [inline test/postman/nr-starting-api_test_collection.postman_collection.json:5]: This file looks like exactly the same as `test/postman/nr-starting-api.postman_collection.json`. Is that correct?

**Review by RMCampos — COMMENTED** [review]
Make sure these files are not the same.

File: test/postman/nr-starting-app_env.postman_environment.json (hunk 1 above)
Comment (dogivalfer) [reply test/postman/nr-starting-app_env.postman_environment.json:1]: I'll further simplify those filenames, but they're supposed to be the same within the same pattern, I'll turn them into something like `starting-api.postman_collection.json` and `starting-api.postman_environment.json`

**Review by RMCampos — APPROVED** [review]
LGTM. Well done!

**Pull request merged**